tensorrt_llm/executor/worker.py

```
@@ -248,6 +249,15 @@ def worker_main(
 
     logger_debug(f"Worker {mpi_rank()} entering worker_main...\n", "green")
 
+    pid = os.getpid()
+    cpus = os.sched_getaffinity(pid)
+    if cpus:
+        logger.warning(
+            f"Found worker process {pid} was bound to {cpus}, this may harm "
+            "performance.", )
+        logger.warning(f"Will clear the cpu affinity")
```

Ruff (0.14.6)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)